lib/views/consignments_slack_view.ex

```diff
@@ -49,7 +49,17 @@ defmodule Aprb.Views.ConsignmentsSlackView do
                         },
                         %{
                           title: "Price In Mind",
-                          value: "#{event["properties"]["minimum_price"]}",
+                          value: event["properties"]["minimum_price"],
+                          short: true
+                        },
+                        %{
+                          title: "Submission ID",
+                          value: event["object"]["id"],
+                          short: true
+                        },
+                        %{
+                          title: "Submission Title",
```

Comment:
My only #small comment is that this should probably go at the top (above `Artist`) and just have the label `Title`. 

Since we're showing this to external partners I think this would make more sense 😄.

But that's my only comment! Otherwise, merge at will.

Reply:
makes sense 🙌